Bottom: 2733a8a302edb8603a30c6319fb86597b0b980bc
Top:    3e66a44426260a8bbd0f5e73db189cbba5cc6d5a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 22:09:27 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 16247a8..385351e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
